niworkflows/func/util.py

```diff
@@ -131,12 +134,12 @@ def init_bold_reference_wf(omp_nthreads, bold_file=None, pre_mask=False,
 
     workflow.connect([
         (inputnode, enhance_and_skullstrip_bold_wf, [('bold_mask', 'inputnode.pre_mask')]),
-        (inputnode, validate, [('bold_file', 'in_file')]),
-        (inputnode, gen_ref, [('sbref_file', 'sbref_file')]),
+        (inputnode, validate, [(('bold_file', ensure_list), 'in_file')]),
+        (inputnode, gen_ref, [(('sbref_file', ensure_list), 'sbref_file')]),
```

Comment:
Definitely don't need this one, since `sbref_file` is now an `InputMultiPath`.

```suggestion
        (inputnode, gen_ref, [('sbref_file', 'sbref_file')]),
```

Speaking of which...

Reply:
🤦‍♂ Ohhh.... I get it now. Thanks!